can_tools/scrapers/official/federal/HHS/common.py

```diff
@@ -0,0 +1,26 @@
+import requests
+
+from typing import Dict
+
+import pandas as pd
+
+from can_tools.scrapers.official.base import FederalDashboard
+
+
+class HHSDataset(FederalDashboard):
```

Comment:
Let's make this inherit from abc.ABC so it is clear it shouldn't be instantiated or run through tests.